lib/assertions.ts

```diff
@@ -24,9 +24,9 @@ export interface ExistsOptions {
   count: number;
 }
 
-export interface Dictionary<T> {
-  [key: string]: T;
-}
+type CSSStyleDeclarationProperty = keyof CSSStyleDeclaration;
+
+type ActualCSSStyleDeclaration = Partial<Record<CSSStyleDeclarationProperty, unknown>>;
```

Comment:
`CSSStyleDeclaration` requires all properties to be defined

Using `Partial` allows the assertions to define a subset of `CSSStyleDeclaration` properties

Using `unknown` here allows us to get around the issue of `CSSStyleDeclaration` having properties like [`readonly length: number;`](https://github.com/microsoft/TypeScript/blob/d3877d294cb2675381ec00e21251ff533d1c71dc/lib/lib.dom.d.ts#L2929) and [`readonly parentRule: CSSRule | null;`](https://github.com/microsoft/TypeScript/blob/d3877d294cb2675381ec00e21251ff533d1c71dc/lib/lib.dom.d.ts#L2999)